### PR TITLE
Replace --skip-dbt with --dbt-test and --dbt-build

### PR DESCRIPTION
**Pull request by even-wei**
Signed-off-by: Even Wei <evenwei@infuseai.io>

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
Enhancement

**What this PR does / why we need it**:
Replace --skip-dbt with --dbt-test and --dbt-build, which eliminates potential performance issues.

**Which issue(s) this PR fixes**:
sc-27024

### CODE REVIEW

File: piperider_cli/cli.py [hunk 1]
```diff
@@ -145,14 +146,18 @@ def run(**kwargs):
     output = kwargs.get('output')
     skip_report = kwargs.get('skip_report')
     skip_recommend = kwargs.get('skip_recommend')
-    skip_dbt = kwargs.get('skip_dbt')
+    run_dbt_test = kwargs.get('dbt_test')
+    run_dbt_build = kwargs.get('dbt_build')
+    dbt_command = 'test' if run_dbt_test else ''
+    if run_dbt_build:
+        dbt_command = 'build'
```
Comment (ctiml) [inline piperider_cli/cli.py:153]: How about using `dbt_command = 'build' if run_dbt_build else 'test' if run_dbt_test else ''`

File: piperider_cli/workspace.py [hunk 2]
```diff
@@ -748,7 +749,7 @@ def _run_dbt_command(table, default_schema, dbt, console):
         return
 
     cmd = dbt.get('cmd', 'test')
-    if cmd not in ['build', 'run', 'test']:
+    if cmd not in ['build', 'test']:
         message = f"'dbt {cmd}' is invalid, only support 'dbt build/run/test'."
```
Comment (ctiml) [inline piperider_cli/workspace.py:753]: We should change the warning message as well. `build/run/test` -> `build/test`

Comment (ctiml) [reply piperider_cli/workspace.py:753]: Or we could remove L751-757 because we check dbt command outside

File: piperider_cli/cli.py (hunk 1 above)
Comment (even-wei) [reply piperider_cli/cli.py]: LGTM

File: piperider_cli/workspace.py (hunk 2 above)
Comment (even-wei) [reply piperider_cli/workspace.py]: Agree, remove it.

**Review by ctiml — APPROVED** [review]
LGTM

**Pull request merged**